### PR TITLE
ZENKO-472 feat: Setup nightly cosbench run

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -78,6 +78,8 @@ models:
       LOCATION_QUOTA_BACKEND: quotabackend
       GCP_BACKEND_SERVICE_KEY: '%(secret:gcp_backend_service_key)s'
       GCP_BACKEND_SERVICE_EMAIL: '%(secret:gcp_backend_service_email)s'
+      COSBENCH_NIGHLTY_SERVICE_ACCOUNT: '%(secret:cosbenchNightlyServiceAccount)s'
```
Comment (JianqinWang) [inline eve/main.yml]: typo? `COSBENCH_NIGHTLY_SERVICE_ACCOUNT` vs `COSBENCH_NIGHLTY_SERVICE_ACCOUNT`, I notice this in line  271 of `tests/Makefile` as well. If this is intentional, what is the reasoning?

File: eve/workers/cosbench/tmpl.awk [hunk 2]
```diff
@@ -0,0 +1,11 @@
+BEGIN{
```
Comment (anurag4DSB) [inline eve/workers/cosbench/tmpl.awk]: This is pretty cool and neat!

**Review by anurag4DSB — APPROVED** [review]

File: eve/workers/cosbench/tmpl.awk [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    values["ACCESS_KEY"] = ak;
+    values["SECRET_KEY"] = sk;
+    values["ENDPOINT"] = ep;
+}
+
+{
+    for (tmpl in values)
```
Comment (anurag4DSB) [inline eve/workers/cosbench/tmpl.awk:8]: This is pretty cool and neat!

File: eve/main.yml (hunk 1 above)
Comment (tmacro) [reply eve/main.yml]: nope thats a typo

**Comment by tmacro** [comment]
@bert-e wait

**Comment by tmacro** [comment]
@bert-e after_pull_request=282

**Comment by tmacro** [comment]
@bert-e after_pull_request=314

File: tests/Makefile [hunk 4]
```diff
@@ -1,3 +1,4 @@
+VERBOSE=True
```
Comment (giacope) [inline tests/Makefile]: Why not `?=`?

Comment (tmacro) [reply tests/Makefile]: ooops was supposed to remove that, its supposed to be unset by default

**Review by giacope — APPROVED** [review]

**Comment by tmacro** [comment]
@bert-e wait

**Review by ssalaues — APPROVED** [review]

**Pull request merged**